scheduler/deploy-to-marathon.sh

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env bash
 curl -k -XDELETE -H "Content-Type: application/json" http://master:8080/v2/apps//elasticsearch-mesos-scheduler?force=true
```

Comment:
At some point we really should stop the assumption that we have master, slave{1..3} records in the hosts file, and use environment variables with [direnv](http://direnv.net) instead :)